roles/goaccess/templates/goaccess-input.service.j2

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Goaccess input stream
+After=caddy.service
+Wants=caddy.service
+
+[Service]
+Type=simple
+User={{ goaccess_user }}
+Group={{ goaccess_user }}
+StandardOutput=socket
+ExecStart=journalctl -f -u caddy.service --output cat --grep http.log.access
```

Comment:
**n.b.** Only access logs must be feed to goaccess, hence I'm creating a pipe with access entries only.